### PR TITLE
Added a parser for Sphinx-build (http://www.sphinx-doc.org)

### PR DESCRIPTION
**Pull request by bobdoah**

### CODE REVIEW

File: src/main/java/hudson/plugins/warnings/parser/SphinxBuildParser.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+     */
+    public SphinxBuildParser() {
+        super(Messages._Warnings_SphinxBuild_ParserName(),
+                Messages._Warnings_SphinxBuild_LinkName(),
+                Messages._Warnings_SphinxBuild_TrendName(),
+                SPHINX_BUILD_WARNING_PATTERN);
+    }
+
+    @Override
+    protected String getId() {
```
Comment (uhafner) [inline src/main/java/hudson/plugins/warnings/parser/SphinxBuildParser.java]: This method should not be overwritten, see JavaDoc.

File: src/test/java/hudson/plugins/warnings/parser/SphinxBuildParserTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+ * Tests the class {@link SphinxBuildParser}.
+ */
+public class SphinxBuildParserTest extends ParserTester {
+    private static final String WARNING_TYPE = Messages._Warnings_SphinxBuild_ParserName().toString(Locale.ENGLISH);
+    private static final String SPHINX_BUILD_ERROR = "ERROR";
+    private static final String SPHINX_BUILD_WARNING = "WARNING";
+
+
+    /**
+     * Parses a file with two SphinxBuild warnings.
```
Comment (uhafner) [inline src/test/java/hudson/plugins/warnings/parser/SphinxBuildParserTest.java]: 6?

**Comment by uhafner** [comment]
Thanks!

**Comment by bobdoah** [comment]
Awesome, no problem.

**Comment by bobdoah** [comment]
@uhafner Is there likely to be a release with this in any time soon?

**Comment by uhafner** [comment]
I'll release it together with https://github.com/jenkinsci/warnings-plugin/pull/83

**Pull request merged**